src/Time/Units.hs

```diff
@@ -49,7 +49,7 @@ import Time.Rational (type (%), type (**), type (//), DivRat, KnownRat, Rat, Rat
 
 import qualified Control.Concurrent as Concurrent
 
-newtype Time (rat :: Rat) = Time RatioNat deriving (Show, Num)
+newtype Time (rat :: Rat) = Time RatioNat deriving (Show, Num, Eq)
```

Comment:
I guess it's also ok to add `Ord`, `Enum`, `Integral`, `Fractional`, `Real` and `RealFrac` instance. Our `Time` newtype is just a wrapper for `Ratio` type so we can everything that `Ratio` has:

* http://hackage.haskell.org/package/base-4.10.1.0/docs/Data-Ratio.html

Maybe even have `Data` and `Storable`, but I don't know how much use we can have in such instances... Let's future issues decide :)

Also, I just discovered, that we don't need `intPart` function. I'm not aware of all numeric functions... But looks like `intPart` [is just `floor` function from `RealFrac` type class](http://hackage.haskell.org/package/base-4.10.1.0/docs/Prelude.html#v:floor). You can try to test it in `ghci` and if so, we can remove `intPart` function :) After deriving `RealFrac`, of course.